app/services/provider/importer.rb

```diff
@@ -26,7 +26,7 @@ def invalid?(provider_attributes)
   end
 
   def fetch_providers(link = nil)
-    accredited_providers = ::AccreditedProvider::Api.call(link:)
+    accredited_providers = ::Provider::Api.call(link:)
```

Comment:
To be consistent, we should rename this to providers instead of accredited_providers. (line 30 as well)
```suggestion
    providers = ::Provider::Api.call(link:)
```

Reply:
resolved in [c8aab49](https://github.com/DFE-Digital/itt-mentor-services/pull/110/commits/c8aab4935ccdfccc6124952ee65eafbea8b40219)